### DISCUSSION TITLE
Allow contributors to claim credit for contributions in the change log entry

### DISCUSSION THREAD

**Discussion by seanbudd** [nvaccess/nvda#15600] [Ideas]
Now that we encourage authors to write change log entries themselves, we are considering allowing contributors to credit themselves for their work.
This would be optional, and would only include authors who submitted the pull request.
The proposal is to allow the pull request author to add their GitHub username to the end of a change log entry.
Additionally, using the GitHub username, we could auto-link their GitHub profile, or their GitHub contributions.

For example, a change log entry could look like:
> * Added support for Bluetooth Low Energy HID Braille displays. ([#15470](https://github.com/nvaccess/nvda/pull/15470), [@alexmoon](https://github.com/nvaccess/nvda/commits?author=alexmoon))

This is to give credit where credit is due, and highlight the contributions that the community makes to NVDA. We are hoping this may also encourage more contributions to NVDA.

Does this proposed implementation make sense?
Contributors: would you do this if it was supported and allowed?

**Comment by nvdaes** [comment]
I will do it to make easy for people to provide feedback to me. Since changelog is translated, people reading changes may choose to contact contributors in their native languages and not just in English, for example in Spanish in my case.

**Comment by codeofdusk** [comment]
I would do this if supported by the project.

**Comment by XLTechie** [comment]
The implementation appears reasonable to me. It seems like a good way to credit 
contributions.
